packages/cli-e2e/utils/terminal/terminal.ts

```diff
@@ -26,21 +30,16 @@ export class Terminal {
     command: string,
     args?: ReadonlyArray<string>,
     options?: SpawnOptionsWithoutStdio,
-    processManager: ProcessManager = new ProcessManager()
+    processManager: ProcessManager = new ProcessManager(),
+    debugName?: string
   ) {
     this.childProcess = processManager.spawn(command, args, options);
-    // TODO CDX-283: Replace console.log by a file logger.
-    if (process.env['COVEO_CLI_E2E_DEBUG']) {
-      this.childProcess.stdout.on('data', (data) =>
-        console.log(`[STDOUT]${data.toString()}`)
-      );
-      this.childProcess.stderr.on('data', (data) =>
-        console.log(`[STDERR]${data.toString()}`)
-      );
-      this.childProcess.stdin.on('data', (data) =>
-        console.log(`[STDIN]${data.toString()}`)
-      );
-    }
+
```

Comment:
I chose to always log, it's dumb to re-run tests because you're missing logs, and it ain't costing much